keras_imaging/datasets/__init__.py

```diff
@@ -0,0 +1,24 @@
+import os.path
+
+import keras.utils.data_utils
+import numpy
+
+
+def load_data(name):
+    hostname = "http://keras-imaging.storage.googleapis.com"
```

Comment:
@sanz999 Write some documentation! You can find an example here:

https://github.com/broadinstitute/keras-rcnn/blob/master/keras_rcnn/datasets/malaria.py#L7-L14